src/test/java/com/programmers/ticketparis/service/schedule/ScheduleServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.programmers.ticketparis.service.schedule;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.programmers.ticketparis.exception.ScheduleException;
+
+@SpringBootTest
+class ScheduleServiceTest {
+
+    @Autowired
+    private ScheduleService scheduleService;
+
+    @ParameterizedTest
+    @DisplayName("존재하지 않는 공연 ID와 스케줄 ID에 대한 스케줄 삭제 시, 예외가 발생한다.")
+    @CsvSource(value = {"0,1", "1,0", "9999999,9999999", "-1,2", "2,-1"})
+    @Transactional
+    void test_delete_schedule_throw_exception(Long performanceId, Long scheduleId) {
+        //when, then
+        assertThatThrownBy(() -> scheduleService.deleteScheduleById(performanceId, scheduleId))
+            .isInstanceOf(ScheduleException.class);
```

Comment:
p2
.hasMessageContaining() 메서드로 예외의 message 내용이 잘 들어갔는지 검증할 수 있습니다! ExceptionRule의 message가 그대로 들어가고 있으니 이대로 추가 해주셔도 될 것 같아요~!
```suggestion
            .isInstanceOf(ScheduleException.class)
            .hasMessageContaining("공연 ID와 스케줄 ID에 해당하는 스케줄이 존재하지 않아 삭제 불가");
```

Reply:
반영: [b3031c6](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/30/commits/b3031c684f87f4839290e4adbd247ff41a119fb6)
- 너무 좋네요! 다만, 메시지를 하드 코딩 하지는 않고 ExceptionRule에 적힌 내용을 불러왔습니다.
- 추가적으로 hasMessage()는 정확히 예외 메시지가 일치하는지를 보기 때문에 정확성을 위해 Containing 대신 hasMessage를 사용했습니다.